package.json

```diff
@@ -21,14 +20,14 @@
     "lint": "peasant lint"
   },
   "dependencies": {
-    "babel-runtime": "^5.5.6",
-    "minim": "^0.14.0",
-    "minim-parse-result": "^0.2.2"
+    "babel-runtime": "^6.23.0",
+    "minim": "^0.15.0",
+    "minim-parse-result": "^0.4.0"
   },
   "devDependencies": {
-    "chai": "^2.3.0",
-    "coveralls": "^2.11.2",
-    "peasant": "^0.5.2"
+    "babel-plugin-add-module-exports": "^0.2.1",
```

Comment:
So, I had to add this so that require('fury') gives a `require('fury').Fury` object. It was doing that by default in babel 5 but not in babel 6. http://stackoverflow.com/questions/33505992/babel-6-changes-how-it-exports-default